aeppl/transforms.py

```diff
@@ -56,25 +54,32 @@ class TransformedRV(RandomVariable, metaclass=TransformedRVMeta):
 
 
 @singledispatch
-def _default_transformed_rv(
+def _default_transformed_rv_op(
```

Comment:
I still think it makes sense to add the `_op` suffix in this commit, as it exists in the related `create_transformed_rv_op`. 

I imagine the original name distinction was exactly because of the different returned types, and that if we were implementing this from scratch we would have chosen similar signatures...